docs/protocol/extension_statistics.md

```diff
@@ -119,6 +120,14 @@ $model_stat =
   due to different batch size (for example, larger batches typically
   take longer to compute).
 
+- "memory_usage" : The memory usage detected during model loading, which may be
```

Comment:
Might not be in scope of this ticket, but I'm curious how this interacts with something like ONNXRuntime with TRT EP. I believe ORT will create new TRT engines for new shapes in inference requests at runtime. Would it be misleading if this reports `A` for the model's memory usage, but is actually taking up `B >> A` after a few requests?